cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-12-15 15:41
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(null=True, upload_to=cinema.models.movie_image_path),
```

Comment:
Ensure that the `cinema.models.movie_image_path` function is defined and correctly implements the logic to format the image filename as `f"{slugify(movie.title)}-{uuid}{ext}"` as specified in the task description. This is crucial for maintaining the correct file naming convention.

Reply:
This is a migration file, it is created automatically. The logic in the model is implemented correctly